apps/api/src/report/report.service.ts

```diff
@@ -221,37 +221,149 @@ export class ReportService {
    */
   async splitReportFourNewOldProcess(f: string, t: string) {
     const statuses = await this.getStatusMap();
-    const entityManager = getManager();
-
     const { from, to } = this.captureFromTo(f, t);
 
     this.logger.log(`getLicensingStageApplicants: Apply date filter from (${from}) and to (${to})`);
+    const connection = getConnection();
+    const mappedStatuses = LIC_REG_STAGE.map(licensingStatus => {
+      return statuses[licensingStatus];
+    });
+    const mappedStatusesString = mappedStatuses.map(status => `'${status}'`).join(',');
+    try {
+      const oldProcess = await connection.query(
```

Comment:
I wonder is using an await Promise.all would be useful here? Quite a few await calls in this section.